src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ReportsControllerTest.java

```diff
@@ -179,7 +180,7 @@ public void should_return_zipfiles_summary_result_in_json_format() throws Except
         mockMvc
             .perform(get("/reports/zip-files-summary?date=2019-01-14&container=bulkscan"))
             .andExpect(status().isOk())
-            .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8_VALUE))
+            .andExpect(content().contentType(APPLICATION_JSON_VALUE))
```

Comment:
it's a different type, isn't it?

Reply:
`APPLICATION_JSON_UTF8_VALUE` is deprecated. Framework suggests to just use `APPLICATION_JSON_VALUE` instead